pkg/networkservice/common/clientinfo/client_test.go

```diff
@@ -100,7 +99,7 @@ func TestLabelsOverwritten(t *testing.T) {
 	client := next.NewNetworkServiceClient(clientinfo.NewClient(), checkrequest.NewClient(t, func(t *testing.T, request *networkservice.NetworkServiceRequest) {
 		assert.Equal(t, expected, request.GetConnection().GetLabels())
 	}))
-	conn, err := client.Request(logger.WithLog(context.Background()), &networkservice.NetworkServiceRequest{
+	conn, err := client.Request(log.WithLog(context.Background()), &networkservice.NetworkServiceRequest{
```

Comment:
I believe there is no need for logger in such place no more, `context.Background` is enough for this.